### DISCUSSION TITLE
完成後刪除原資料夾

### DISCUSSION THREAD

**Discussion by Ian-Lin8239** [Yuukiy/JavSP#209] [想法 | Ideas]
開發者你好
使用您的刮削器非常方便幫我整理大量影片庫，能否多個功能是刮削完成後把原始資料夾一併刪除呢？

**Comment by Yuukiy** [comment]
感谢支持。之前没有做这个功能主要是担心误删文件夹，假设是从网上下载的影片，整理过后原位置下剩下的大概率是广告，但是如果有正常视频文件被识别为广告，可能会导致视频被误删

**Reply by Ian-Lin8239** [reply]
確實有此風險，感謝您的回覆！

**Comment by qicfan** [comment]
其实可以把刮削完成后的空文件夹删除

**Comment by Leelagozero** [comment]
总之，你不能随意动和改变人家原来的东西，你可以生成自己的。这个是原则。但是。。。。你这个软件！
